builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -291,24 +292,16 @@ node() {
 
 							case 'gcp_apigee':
 								events.sendStartedEvent('DEPLOY_TO_GCP_APIGEE', 'Deployment started for APIGEE', context_map, environment_logical_id)
-								def arn = awsLambdaModule.getLambdaARN(stackName)
 								apiHostName ="${configLoader.APIGEE.API_ENDPOINTS[env_key].service_hostname}"
 								updateSwagger(apiHostName, environment_logical_id, current_environment, env_key, config)
-								endpointUrl = apigeeModule.deploy("swagger/swagger.json", arn, env_key, environment_logical_id, config, context_map)
+								endpointUrl = apigeeModule.deploy("swagger/swagger.json", lambdaARN, env_key, environment_logical_id, config, context_map)
 								echo "API Proxy created at: ${endpointUrl}"
-                events.sendCompletedEvent('CREATE_ASSET', null, utilModule.generateAssetMap("gcp", null, "apigee", config), environment_logical_id);
 								events.sendCompletedEvent('DEPLOY_TO_GCP_APIGEE', 'Deployment complete for APIGEE', context_map, environment_logical_id)
+                events.sendCompletedEvent('CREATE_ASSET', null, utilModule.generateAssetMap("gcp", endpointUrl, "apigee", config), environment_logical_id);
```

Comment:
Asset type should be endpoint_url  instead of apigee (since you are passing endpointUrl as value).

Reply:
i am passing enpoint_url as it needs some value and i can't pass arn. Passing null to provider_id will throw an error and not create the asset.

Reply:
updated